hasura/metadata/databases/default/tables/public_app.yaml

```diff
@@ -77,6 +77,14 @@ select_permissions:
       filter:
         team_id:
           _eq: X-Hasura-Team-Id
+  - role: reviewer
```

Comment:
is this needed? considering the backend should connect to Hasura using the service role

Reply:
Moving here but I think it's better if we generate a client with reviewer permissions to perform the verify since we can be more granular with permissions. If we use the service client we will need to add more permissions to service. The backend is currently using the reviewer role since the `const reviewer_client = await getAPIReviewerGraphqlClient();` call is set with a reviewer role header variable